vmas/scenarios/navigation_obstacles.py

```diff
@@ -395,17 +396,17 @@ def observation(self, agent: Agent):
         )
 
     def done(self):
```

Comment:
before this change, VMAS would trigger a done() on the env before the final reward is ever applied